controllers/feature/controller.go

```diff
@@ -175,3 +172,18 @@ func (f *featureHandler) isEnabled(name string) bool {
 	_, ok := f.featureState[name]
 	return ok
 }
+
+func (f *featureHandler) getDepFeature(depName, namespace string) (*v1.Feature, error) {
+	start := time.Millisecond * 250
```

Comment:
why is this needed?

Reply:
This is just to avoid `not found` error. The first try will always return `not found` error because the race condition when feature is not actually created.